run-celery-local.sh

```
@@ -0,0 +1,37 @@
+# Make sure Redis is running
+echo "Checking if Redis is running..."
+if ! docker-compose -f docker-compose.local.yml ps | grep -q "cache.*Up"; then
+    echo "Redis is not running. Starting required Docker services..."
+    docker-compose -f docker-compose.local.yml up -d cache
+    sleep 5
+fi
+
+# Activate virtual environment from .venv folder instead of venv
+source .venv/bin/activate
+
+cd api
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)